src/Repository/EventRepository.php

```diff
@@ -111,6 +113,53 @@ public function findOneActiveBySlug(string $slug): ?BaseEvent
         ;
     }
 
+    public function findStartedEventBetweenDatesForTags(
+        \DateTimeInterface $startDate,
+        \DateTimeInterface $endDate,
+        array $referentTags
+    ): array {
+        if (!$referentTags) {
+            return [];
+        }
+
+        $qb = $this
+            ->createQueryBuilder('event')
+            ->addSelect('adherent')
+            ->join('event.organizer', 'adherent')
+            ->where('event.beginAt < :end_date AND event.finishAt > :start_date')
+            ->andWhere('event.status = :status')
+            ->setParameters([
+                'start_date' => $startDate,
+                'end_date' => $endDate,
+                'status' => Event::STATUS_SCHEDULED,
+            ])
+        ;
+
+        if (Event::class === $this->getEntityName()) {
```

Comment:
Referent events do not have associated committee. With this condition you ignore the referent events, they will not be canceled.

Reply:
oh thanks for that, fixed!